application/logging_default_config.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+disable_existing_loggers: False
```

Comment:
Since we're in charge of all the logging config in this project and we do not need the logs from our dependencies, why don't we set this to `True`?